src/common/components/elements/ModelViewer.tsx

```diff
@@ -3,7 +3,7 @@ import '@google/model-viewer';
 
 const ModelViewer = (props: any) => (
   // @ts-ignore
-  <model-viewer {...props} style={{ width: 800, height: 400 }}></model-viewer>
+  <model-viewer {...props} style={{ width: '60vw', height: '30vw' }}></model-viewer>
```

Comment:
I'd like to see this be width: 100% and then define some width based media queries to adjust the height.

Reply:
We want the preview modal to take the full width of the viewport?

Reply:
So we want the width to be 100% and then set the width directly on the modal that contains it?  This is a bit trickier because of the cascade and needing to target multiple antd classnames in the modal.  I can move out the explicit settings up to to where the component is called and keep this component non-opinionated.